templates/wordpress/1/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+services:
+  mariadb:
+      image: 'bitnami/mariadb:latest'
+      volumes:
+        - 'mariadb_data:/bitnami'
+      environment:
+        - MARIADB_USER=bn_wordpress
+        - MARIADB_DATABASE=bitnami_wordpress
+        - ALLOW_EMPTY_PASSWORD=yes
+  wordpress:
+      image: 'bitnami/wordpress:latest'
+      ports:
+        - '${public_port}:80'
+      volumes:
+        - 'wordpress_data:/bitnami'
+      depends_on:
+        - mariadb
+      environment:
+        - MARIADB_HOST=mariadb
+        - MARIADB_PORT_NUMBER=3306
+        - WORDPRESS_DATABASE_USER=bn_wordpress
+        - WORDPRESS_DATABASE_NAME=bitnami_wordpress
+        - ALLOW_EMPTY_PASSWORD=yes
```

Comment:
All the env info is hardcoded, please use questions in the rancher-compose, with these as default values.